CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2258,7 +2327,11 @@ end
 ----------------
 -- Scroll bar --
 ----------------
-function Methods.ApplyScrollbarToFrame(devConsole, scrollbar, window, body, frame)
+function Methods.ApplyScrollbarToFrame(devConsole, 
```

Comment:
I know this code has been here for awhile, but I'm curious what it does in general? Is this a Lua implementation of ScrollingFrame?